projects/cds/src/default-cds-config.ts

```diff
@@ -0,0 +1,18 @@
+import { CdsConfig } from './cds-config';
+
+export const defaultCdsConfig: CdsConfig = {
+  cds: {
+    tenant: 'argotest',
```

Comment:
I understand that these values are for testing / development, but should any of these values be default for a customer?

Reply:
Maybe some values that might not need to end up being defaults for customers can be documented in the README mentioned here: https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5298/files?file-filters%5B%5D=.html&file-filters%5B%5D=.ts#diff-2395cab4c197efc5d8bc7f3525e7e404R82

Reply:
blanked